application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -9,11 +9,10 @@
  */
 enum UserStrings {
     NEW_QUESTION("""
-            Thank you for asking a question in an available channel.
-            When a helper who can answer this question reads it they will help you.
+            Thank you for asking in an available channel. A helper will be with you shortly!
             Please be patient.
```

Comment:
I feel like 'Please be patient' is redundant.

We have:
> A helper will be with you shortly!

and,

> Do not post your question in other channels.

I think our message is clear. Without it, it will be shorter and cleaner, and maybe less aggressive when we don't repeat the same thing 10 times. What do you think?